ulid.gemspec

```diff
@@ -17,5 +17,5 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ['lib']
 
-  spec.add_dependency 'sysrandom', '>= 1.0.0', '< 2.0'
+  spec.required_ruby_version = '>= 2.5.0'
```

Comment:
SecureRandom exists even in https://www.rubydoc.info/stdlib/securerandom/1.8.7/SecureRandom

so we don't need to specify version at all

Reply:
Correct me if I'm wrong but my understanding about the reason of adopting sysrandom gem in ulid is that [SecureRandom does not use /dev/urandom first](https://bugs.ruby-lang.org/issues/9569) before 2.5. In 2.5+ [it does](https://bugs.ruby-lang.org/projects/ruby-trunk/repository/trunk/revisions/57384).

Reply:
so we can leave this dependency with constrain:
`spec.add_dependency 'sysrandom', '>= 1.0.0', '< 2.0' if RUBY_VERSION < "2.5"`

Reply:
hmm I am not sure if we can do it in gemspec ; )

Reply:
with my commit 09e7737, now sysrandom is required if necessary.

With Ruby 2.4.3 (before 2.5.0)

```
GIT
  remote: https://github.com/sakuro/ulid
  revision: 09e7737febfbefa20dd78acbf655674fb9473d7c
  branch: remove-sysrandom
  specs:
    ulid (1.0.0)
      sysrandom (>= 1.0.0, < 2.0)

GEM
  remote: https://rubygems.org/
  specs:
    sysrandom (1.0.5)

PLATFORMS
  ruby

DEPENDENCIES
  ulid!

BUNDLED WITH
   2.0.1
```

With Ruby 2.6.1 (latest)

```
GIT
  remote: https://github.com/sakuro/ulid
  revision: 09e7737febfbefa20dd78acbf655674fb9473d7c
  branch: remove-sysrandom
  specs:
    ulid (1.0.0)

GEM
  remote: https://rubygems.org/
  specs:

PLATFORMS
  ruby

DEPENDENCIES
  ulid!

BUNDLED WITH
   2.0.1
```

Reply:
Looks good to me 👍 

(you can update PR title + description)

Reply:
Updated title and desc.